qa/tasks/cephfs/kernel_mount.py

```diff
@@ -94,6 +94,7 @@ def umount(self, force=False):
 
         self.mounted = False
         self.cleanup()
+        super(KernelMount, self).umount()
```

Comment:
This looks unrelated to the PR. Please have a separate commit explaining.

Reply:
Will add the kwargs instead and remove the unrelated code in this PR.